sputnikdao2/README.md

```diff
@@ -134,43 +271,274 @@ ProposalKind::AddBounty { .. } => "add_bounty",
-RemoveProposal,
-/// Vote to approve given proposal or bounty.
-VoteApprove,
-/// Vote to reject given proposal or bounty.
-VoteReject,
-/// Vote to remove given proposal or bounty (because it's spam).
-VoteRemove,
-/// Finalize proposal, called when it's expired to return the funds
-/// (or in the future can be used for early proposal closure).
-Finalize,
-/// Move a proposal to the hub to shift into another DAO.
-MoveToHub
-```
+> DAO votes to select some token to become voting token (only can be done once, can't change later).
+
+User flow is next:
+
+- User's deposit the token into the DAO.
+- They can then choose who to delegate these tokens. It can be to themself or to other users to increase their vote weight.
+- When users vote for proposals, their vote is weighted by all the delegations to them.
+- Undelegating will block delegating / withdrawing until one voting period passes.
+- Undelegated tokens can be withdrawn by the user.
+
+---
 
-so, for example a role with: `["mint:VoteReject","mint:VoteRemove"]` means the users with that role can only vote to *reject or remove a mint proposal*, but they can't vote to approve.
+## Bounties
+
+> Add and configure bounties using `AddBounty` proposal.
+
+The lifecycle of a bounty is the next:
+
+- Anyone with permission can add proposal `AddBounty` which contains the bounty information including `token` to pay the reward in and `amount` to pay it out.
+- This proposal gets voted in by the current voting policy.
+- After proposal is passed, the bounty gets added. Now it has an `id` in the bounty list which can be queried via `get_bounties`.
+- Anyone can claim a bounty by calling `bounty_claim(id, deadline)` up to `repeat` times which was specified in the bounty. This allows to have repetitive bounties or multiple working collaboratively.
+- `deadline` specifies how long it will take the sender to complete the bounty.
+- If claimer decides to give up, they can call `bounty_giveup(id)`, and within `forgiveness_period` their claim bond will be returned. After this period, their bond is forfeited and is kept in the DAO.
+- When a bounty is complete, call `bounty_done(id)`, which will add a proposal `BountyDone` that, when voted, will pay to whoever completed the bounty.
 
-You can use `*` as a wildcard, so for example a role with `mint:*` can perform any vote action on mint proposals.
+---
 
-You can also use `*:*` for unlimited permission, normally the `council` role has `*:*` as its configured permission so they can perform any vote action on any kind of proposal.
+## Blob storage
+
+> DAO supports storing larger blobs of data and content indexing them by hash of the data. This is done to allow upgrading the DAO itself and other contracts.
+
+Blob lifecycle:
+
+- Store blob in the DAO.
+- Create upgradability proposal.
+- Proposal passes or fails.
+- Remove blob and receive funds locked for storage back.
+
+Blob can be removed only by the original storer.
+
+---
+
+```
+
```

Comment:
Since we are going to add new upgradability logic (https://github.com/near-daos/sputnik-dao-contract/pull/69/files), this example will no longer be accurate.

I think it's best to skip the example for now and provide one when the new upgradability logic gets merged.

Reply:
If you were referring to the `examples` section, the `proposals` sections already covers some examples. It's good to have, but not mandatory. We can create a separate PRs just for the examples.